src/frontend/src/common/queries/samples.ts

```diff
@@ -271,7 +277,14 @@ export function useCreateSamples({
  * sample cache
  */
 const mapSampleData = (data: SampleResponse) => {
-  return reduceObjectArrayToLookupDict<Sample>(data.samples, "publicId");
+  const { samples } = data;
```

Comment:
do you need to call `camelize` on `samples` here?

Reply:
No, because samples is fetch via `apiResponse`, which already calls `camelize`